src/ctap/ctap1.rs

```diff
@@ -399,15 +393,19 @@ mod test {
         let mut env = TestEnv::new();
         env.user_presence()
             .set(|_| panic!("Unexpected user presence check in CTAP1"));
-        let mut ctap_state = CtapState::new(&mut env, START_CLOCK_VALUE);
+        let mut ctap_state = CtapState::new(&mut env, CtapInstant::new(0_u64));
```

Comment:
I don't see why we need to write `0_u64` here and not in other files. It seems to compile just fine with `0`.

Reply:
Done.